packages/jbrowse-plugin-apollo/src/components/ImportFeatures.tsx

```diff
@@ -0,0 +1,177 @@
+    }
+  }, [])
+
+  const getAssemblies = () => {
+    const uri = new URL('/assemblies', baseURL).href
+    const apolloFetch = apolloInternetAccount?.getFetcher({
+      locationType: 'UriLocation',
+      uri,
+    })
+    if (apolloFetch) {
+      apolloFetch(uri, {
+        method: 'GET',
+      })
+        .then((response) => response.json())
+        .then((res) => {
+          console.log(`Found assemblies: ${JSON.stringify(res)}`)
+          res.forEach((item: Collection) => {
+            setCollection((result) => [
+              ...result,
+              {
+                _id: item._id,
+                name: item.name,
+              },
+            ])
+          })
+        })
+    }
+  }
+
+  async function onSubmit(event: React.FormEvent<HTMLFormElement>) {
+    event.preventDefault()
+    let fileChecksum = ''
+    const sessionToken = sessionStorage.getItem('apolloInternetAccount-token')
+    if (sessionToken == null) {
+      alert('You must authenticate first!')
+      return
+    }
+
+    // First upload file
+    const url = new URL('/files', baseURL).href
+    const formData = new FormData()
+    formData.append('file', file)
+    formData.append('fileName', file.name)
+    formData.append('type', 'text/x-gff3')
+    const apolloFetchFile = apolloInternetAccount?.getFetcher({
+      locationType: 'UriLocation',
+      uri: url,
+    })
+    if (apolloFetchFile) {
+      const res = await apolloFetchFile(url, {
+        method: 'POST',
+        body: formData,
+      })
+      console.log(`File upload response is ${res.status}`)
+      if (res.ok) {
+        fileChecksum = (await res.json()).checksum
+      } else {
+        throw new Error(
+          `Error when inserting new features (while file uploading): ${res.status}, ${res.text}`,
+        )
+      }
+    }
+    console.log(`File uploaded, file checksum "${fileChecksum}"`)
+    console.log(`AssemblyId is "${assemblyId}"`)
+    const res = await fetch(new URL('/changes/submitChange', baseURL).href, {
+      method: 'POST',
+      body: JSON.stringify({
+        changedIds: ['1'],
+        typeName: 'AddFeaturesFromFileChange',
+        assemblyId,
+        fileChecksum,
+        assemblyName,
+      }),
+      headers: new Headers({
+        Authorization: `Bearer ${sessionToken}`,
+        'Content-Type': 'application/json',
+      }),
+    })
+    console.log(`Adding features response is ${res.status}`)
+    if (res.ok) {
+      alert('Features added succesfully!')
+    } else {
+      throw new Error(
+        `Error when inserting new features: ${res.status}, ${res.text}`,
+      )
+    }
+    handleClose()
+    event.preventDefault()
+  }
+
+  return (
+    <Dialog open maxWidth="xl" data-testid="login-apollo">
+      <DialogTitle>Import Features from GFF3 file</DialogTitle>
+      <form onSubmit={onSubmit}>
+        <DialogContent style={{ display: 'flex', flexDirection: 'column' }}>
+          <h4>Select assembly</h4>
```

Comment:
Use `DialogContentText` instead of `h4` (see [here](https://v4.mui.com/components/dialogs/) for examples)

Reply:
done